docs/examples/notebooks/learn/TestStatistics.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>I think this could use with some expansion of the code in the <code>qmu</code>  functions. Additionally, maybe adding a comment, and fixing up some variable names. Would also be helpful to have the citation of the formulae again (and perhaps including the mathematical formulae).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scikit-hep/pyhf/pull/1014/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>